custom_components/tuya_local/devices/ion_wifi_dimmer.yaml

```diff
@@ -0,0 +1,58 @@
+name: ION WIFI DIMMER
+products:
+  - id: ceikasvm5p5ymcax
+    name: ION WIFI DIMMER
+primary_entity:
+  entity: light
+  dps:
+    - id: 1
+      name: switch
+      type: boolean
+    - id: 2
+      name: brightness
+      type: integer
+      range:
+        min: 10
+        max: 1000
+secondary_entities:
+  - entity: number
+    name: Minimum brightness
+    category: config
+    icon: "mdi:brightness-percent"
+    dps:
+      - id: 3
+        type: integer
+        name: value
+        range:
+          min: 10
+          max: 1000
+        mapping:
```

Comment:
No mapping is required, as step 1 is the default.

Reply:
Removed mapping with default step value 1 as this is the default value.